dev/io.openliberty.org.jboss.resteasy.common.ee10/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
is `https://jakarta.ee/xml/ns/jakartaee/` the correct xmlns?

Reply:
I believe it would look something like:
```
<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns="https://jakarta.ee/xml/ns/jakartaee"
       xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee 
        https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd"
       version="3.0"
       bean-discovery-mode="annotated">
</beans>
```